Documentation/mobile-manual.txt

```diff
@@ -114,6 +114,25 @@ information is stored on the cloud server and access to the cloud is enabled. In
 the dive list is initially empty. Start entering dives in the dive
 log or download dive information from a supported dive computer.
 
+****
+[icon="images/icons/important.png"]
+[IMPORTANT]
+*Cloud problems: FAQ*
+
+[blue]#After filling in the correct information and submitting it, I did not receive a PIN by email#
+
+1. Check *carefully* that there is not a spelling mistake in the email address you provided.
+2. Check that the return email containing the PIN is not in the SPAM folder of your email client.
+3. Wait for 24 hours and check again for an email with a PIN. 
+4. If there is still no email containing a PIN, send mail to subsurface@subsurface-divelog.org
```

Comment:
since that is the developer mailing list (and it rejects email from unknown senders), maybe just have them send email to `in-app-support@subsurface-divelog.org` -- those emails go only to me (and right now I'm typically the only one able to help, anyway).